### PR TITLE
Require `.form-label` classes on form labels

### PR DESCRIPTION
**Pull request by MartijnCuppens**
There's currently no clean way to theme form labels, simply because they have no class. By just adding styles to the `<label>` element, styles can leak to other elements like the checkbox labels.

I added a new `.form-label` class and some `null` variables which makes it easier to theme these labels and prevent style leaks.

### CODE REVIEW

**Review by ffoodd — APPROVED** [review]

**Comment by MartijnCuppens** [comment]
Gonna wait for @mdo's approval before merging this.

**Review by mdo — APPROVED** [review]
I've debated this for the last two major versions, so I'm definitely inclined to make the switch. Styling all `<label>`s led to a few extra lines, so making that update feels like a promising move for others.

**Pull request merged**
